independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,569 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HubSpot CMS",
+    "description": "HubSpot CMS community connector.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api.hubapi.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/content/api/v2/pages": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "List pages (V2)",
+        "description": "Get all of your pages. Supports paging and filtering.",
+        "operationId": "PagesList",
+        "parameters": [
+          {
+            "name": "limit",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "default": 20,
+            "description": "Limit",
+            "x-ms-summary": "limit",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "archived",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "default": false,
+            "enum": [
+              false,
+              true
+            ],
+            "description": "Archived",
+            "x-ms-summary": "archived",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "id",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "ID",
+            "x-ms-summary": "id",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Name",
+            "x-ms-summary": "name",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create a new page (V2)",
+        "description": "Create a new page.",
+        "operationId": "PagesCreate",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "campaign": {
+                  "title": "Campaign",
+                  "type": "string",
+                  "description": "campaign"
+                },
+                "campaign_name": {
+                  "title": "Campaign name",
+                  "type": "string",
+                  "description": "campaign_name"

```

Comment:
@troystaylor Can you please provide valid description valid with out grammatical values.

Documentation - [Description of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar values for the properties in swagger.